packages/cli/src/commands/RunProjectCommand.ts

```diff
@@ -21,7 +21,7 @@ export class RunProjectCommand<T> implements CommandModule<T, RunProjectArgs> {
             default: 'start',
             describe: 'Allows using different NPM script than `start`, e.g. `crawlee run --script=start:prod`.',
         });
-        return args as Argv<RunProjectArgs>;
+        return args as unknown as Argv<RunProjectArgs>;
```

Comment:
we will need to update TS in website folder too, otherwise typedoc wont work. I got it already tested elsewhere, it seems to work fine with TS 4.8.

if its just about this one line, we can do this as well, but if we bump the version in docs i think it will work without it